util/utility.js

```diff
@@ -31,7 +31,7 @@ function betterFormatting(i) {
 }
 
 function removeFormatting(i) {
-  return i.replace(/§./g, '');
+  return i.replace(/§.|⚚|✦|✪/g, '').trim();
```

Comment:
This function is used elsewhere, so you should probably move these to line 34 on `processSkyBlock`